azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/artifacthandler/ZIPArtifactHandlerImpl.java

```diff
@@ -25,7 +27,21 @@ public void publish(DeployTarget target) throws MojoExecutionException, IOExcept
         prepareResources();
         assureStagingDirectoryNotEmpty();
 
-        target.zipDeploy(getZipFile());
+        final File zipFile = getZipFile();
+        int retryCount = 0;
+
```

Comment:
remove this empty line here since the `retryCount` has a strong relationship with the while loop.